args.go

```diff
@@ -146,6 +146,8 @@ const (
 	ArgTagName = "tag-name"
```

Comment:
Please add comments to `ArgTagName` and `ArgTagNames` marking them as deprecated in favor of `ArgTag`. Feel free to include (as in, please include) a pithy summary of the situation with the deprecation statement. kthxbai